R/utils.R

```diff
@@ -86,3 +86,15 @@ null_to_NA_list <- function(list) {
 
   return(list)
 }
+
+#' Generate a SHA256 hash file from a downloaded WHO data file
+#'
+#' @param list A data object
+#' @return A character string of length 64 containing the SHA256 string
+create_hash_from_data <- function(data)
+{
+  return( digest::digest(data, algo="sha256", serialize=T) )
```

Comment:
Little nitpicking but for consistency: let's try to follow the style guide by Wickham (http://adv-r.had.co.nz/Style.html), except for spaces between function arguments, e.g., algo="sha256" instead of algo = "sha256". In this case I would suggest rewriting the function as:

create_hash_from_data <- function(data) {
   return(digest::digest(data, algo="sha256", serialize=T))
}